cli/src/mcp-server.ts

```
@@ -1,559 +1,201 @@
-    );
+  }
 
-    this.setupHandlers();
+  /**
+   * Start MCP server using CLI configuration
+   */
+  async start(options: MCPServerOptions = {}): Promise<void> {
+    await this.config.init();
+
+    const {
+      mode = 'stdio',
```

ESLint

[error] 'mode' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)